### ISSUE TITLE
GPT: Launch an Ansible Job thru AAP

### ISSUE THREAD

**Issue by serenamarie125** [janus-idp/software-templates#127]
## Goal
_Details to follow_

### What problem does this solve?
_Details to follow_
 
### Use cases
_Details to follow_

### Acceptance criteria
_Details to follow_

## Issues in Epic
- https://github.com/janus-idp/backstage-showcase/issues/383
- https://github.com/janus-idp/software-templates/issues/152 [Doc]
- https://github.com/janus-idp/software-templates/issues/153 [QE]

**Comment by rtaniwa** [comment]
@jasperchui assigning to you. Ryan Cook has the PRs up for engineering work and I have linked. Once the last one is merged, we can just close with QE and Doc.

**Comment by rtaniwa** [comment]
Going to keep this open until QE and Doc work is complete.

**Comment by kadel** [comment]
As discussed during the triage call, this can be shipped independently from Milestone 3 or Milestone 4.

There is also the outstanding question of how this is different from #128 (what is the difference between launching and creating an ansible playbook in the context of the Backstage template?)

**Comment by jasperchui** [comment]
No longer needed according to https://github.com/janus-idp/software-templates/issues/193
